contrib/ci/configure.sh

```diff
@@ -82,7 +82,8 @@ if [[ "$DISTRO_BRANCH" == -redhat-fedora-3[5-9]* ||
     )
 fi
 
-if [[ "$DISTRO_BRANCH" == -redhat-fedora-* ]]; then
+if [[ "$DISTRO_BRANCH" == -redhat-fedora-* ||
+      "$DISTRO_BRANCH" == -redhat-centos-[9-10]*- ]]; then
```

Comment:
There is an error in the expression. maybe it would be better to match both versions in separate lines.

Reply:
Done